Formula/c/cfn-lint.rb

```diff
@@ -3,10 +3,9 @@ class CfnLint < Formula
 
   desc "Validate CloudFormation templates against the CloudFormation spec"
   homepage "https://github.com/aws-cloudformation/cfn-lint/"
-  url "https://files.pythonhosted.org/packages/ec/bd/60a3f36b698caeec8f4e7e35e030e268c24701de4dcccd07a5e5e1626827/cfn-lint-0.81.0.tar.gz"
-  sha256 "532cbfe076fa8b68e70ec67743e9086169ef6d15be3306cae8aa57b38994fd8f"
+  url "https://files.pythonhosted.org/packages/f8/21/0532999b98a47d2cb0cc987506ea87f79b024433cbc2f80e3c87d7d60da2/cfn-lint-0.82.0.tar.gz"
```

Comment:
0.82.1 is out with quick fix for https://github.com/aws-cloudformation/cfn-lint/issues/2911 so may as well update to that.

Reply:
👍